napari/layers/points/points.py

```diff
@@ -213,6 +209,8 @@ class Points(Layer):
         of the specified property that are mapped to 0 and 1, respectively.
         The default value is None. If set the none, the clims will be set to
         (property.min(), property.max())
+    current_symbol : Symbol
+        Symbol for the next point to be added or the currently selected points.
```

Comment:
The `Symbol` is defined in a private module. I expect it will not be visible when reading documentation generated from API?  If we would like to use it in documentation then it should be part of public API. Otherwise, we should list here al options as const literals.

Reply:
This is the same as we currently have for thing like `Mode`: they are defined in a private module but used publicly. The options are listed in the layer parameters docstring.